pkg/controller/controller_instance.go

```diff
@@ -1192,7 +1192,7 @@ func (c *controller) resolveReferences(instance *v1beta1.ServiceInstance) (bool,
 		return c.resolveNamespacedReferences(instance)
 	}
 
-	return false, stderrors.New(errorAmbiguousPlanReferenceScope)
+	return false, fmt.Errorf("%v", errorAmbiguousPlanReferenceScope)
```

Comment:
What this change?

Reply:
the content variable errorAmbiguousPlanReferenceScope is "Couldn't determine if the instance refers to a Cluster or Namespaced ServiceClass/Plan". golint says that "Error string should not be capitalized or end with punctuation".

Reply:
Hmm, doesn't this change then just hide it from the linter? Why don't you simply change the error message to lowercase?
And what's odd is that golint on my machine doesn't complain at all.

Reply:
i saw the warning in goland, when i open it in vscode, the warning missed. i was confused. 
so i change the contenct to lowercase.